src/expmv_fun.jl

```diff
@@ -41,22 +41,22 @@ function expmv(t, A, b; M = [], prec = "double", shift = false, full_term = fals
     end
     
     if isempty(M)
-        tt = 1
+        tt = 1.0
         M = select_taylor_degree(t * A, size(b, 2))
     else
         tt = t
     end
     
-    tol =   
-      if prec == "double"
-          2.0^(-53)
+    tol =
+      if prec == "half"
```

Comment:
While it won't affect performance really, I think to match common Julia syntax these should be symbols instead of strings.